docs/expert/build.md

```diff
@@ -27,7 +27,7 @@ The following dependencies must be installed:
 - [Node.js](https://nodejs.org/) (installing via [nvm](https://github.com/nvm-sh/nvm) is suggested)
 - [protobuf](https://github.com/protocolbuffers/protobuf)
 - [Python 3](https://www.python.org/) (or newer)
-- [Java 11](https://www.java.com/) (or newer)
+- [Java 11](https://www.java.com/)
```

Comment:
Why removing "or newer"? I'm currently using Java 17 and it works just fine.

Reply:
Some of the tools dont compile without java 11

Reply:
This has been addressed in 4731196f36bfcd3b237dc55af34579da35f8e0e9 by removing smali in the project. @iDrinkCoffee-TG fixed this in 8fcf3320e6d4a373d5d8ea9c01b31bc8363930e4 but I'm waiting for you to confirm it is okay

Reply:
Will test on my end once again

Reply:
marking as resolved, since this works fine now